workflows/dags/cds/cds_harvest.py

```diff
@@ -0,0 +1,140 @@
+import datetime
+import logging
+
+from airflow.decorators import dag, task, task_group
+from airflow.exceptions import AirflowException, AirflowSkipException
+from airflow.macros import ds_add
+from airflow.models.param import Param
+from hooks.generic_http_hook import GenericHttpHook
+from hooks.inspirehep.inspire_http_record_management_hook import (
+    InspireHTTPRecordManagementHook,
+)
+from include.utils.alerts import task_failure_alert
+from include.utils.cds import get_record_for_provided_ids, has_any_id
+from inspire_utils.record import get_value, get_values_for_schema
+
+logger = logging.getLogger(__name__)
+
+
+@dag(
+    start_date=datetime.datetime(2025, 5, 22),
+    schedule="@daily",
+    catchup=False,
+    tags=["cds"],
+    params={
+        "since": Param(type=["null", "string"], default=""),
+    },
+    on_failure_callback=task_failure_alert,
+)
+def cds_harvest_dag():
+    """Defines the DAG for the CDS harvest workflow."""
+    generic_http_hook = GenericHttpHook(http_conn_id="cds_connection")
+    inspire_http_record_management_hook = InspireHTTPRecordManagementHook()
+
+    @task(task_id="get_cds_data")
+    def get_cds_data(**context):
+        """Starts harvesting cds.
+
+        Returns: data from cds
+        """
+        ds = context["ds"]
+        since = context["params"]["since"] or ds_add(ds, -1)
```

Comment:
@drjova is it enough if `since` is not provided to fallback to previous day ? We can also have 
`
since = context["params"]["since"] or context.get("prev_execution_date_success") or ds_add(ds, -1)
`
which fallsback first to the last successful run

Reply:
we can just `-1`